package.json

```diff
@@ -3,6 +3,12 @@
   "version": "2.0.0",
   "description": "",
   "main": "index.d.ts",
+  "files": [
+    "index.d.ts",
```

Comment:
Minor: `index.d.ts` will be included anyway since it is specified in the `main` property